### DISCUSSION TITLE
Regarding removing a CSS class that has been added through dataview metadata

### DISCUSSION THREAD

**Discussion by Gbhuvanesh26** [ghiscoding/Angular-Slickgrid#906] [Q&A]
Hi, 
In slick grid rows to make rows differentiable i added a class through data view metadata, now when I have to reset and remove the classes added so the grid can be moved back to its initial state is there a way to do it ?? I tried dataview.setitems, grid.invalidateallrows and render, but these didn't help. Please let me know if there is any way to reset grid to its initial state with initial css classes. thanks in advance.

**Comment by ghiscoding** [comment]
I don't fully understand your question, but you can change css via metadata as shown in [Example 11](https://ghiscoding.github.io/Angular-Slickgrid/#/additem), the last button changes some rows to light purple background. [Grid State](https://github.com/ghiscoding/Angular-Slickgrid/wiki/Grid-State-&-Preset#example) can give you the `headerCssClass` but not the metadata, you can read it via the code in Example 11 and/or see this [Stack Overflow](https://stackoverflow.com/questions/14514999/add-css-style-to-edited-cell-in-slickgrid) about it. There is also this other Wiki that I wrote, [Dynamically Add CSS Classes to Item Rows](https://github.com/ghiscoding/Angular-Slickgrid/wiki/Dynamically-Add-CSS-Classes-to-Item-Rows)

Please ask your question on Stack Overflow where a lot more users can help answering such general questions
